### PR TITLE
Cp 92 checklist

### PR DESCRIPTION
**Pull request by adamstag20**
Added end of level displays for success and fail of each level. Confetti for success. Would need to build new component for mapping more objectives as they come.

### CODE REVIEW

File: Isle of Trust/src/components/EndOfLevel.tsx [hunk 1]
```diff
@@ -1,35 +1,66 @@
...
+                                <button onClick={updateLevel}> Next Level</button>
+                            </Link>
+                        <Confetti width={width} height={height}/>
+                        </div>
+
+                ) : (
+                    <div className="popup-content">
+                        <h1 className="text-center">Oh No!</h1>
+                        <h2 className="fail">Level {level} Failed</h2>
+                        <h2 className="text-center"> some Players were unable to pay their mortgage!</h2>
```
Comment (hollan2) [inline Isle of Trust/src/components/EndOfLevel.tsx:48]: can we make Players lowercase here

File: Isle of Trust/src/components/EndOfLevel.tsx [hunk 2]
```diff
@@ -1,35 +1,66 @@
...
-                    <Link
-                        reloadDocument
-                        className="link"
-                        to={"/level" + (level + 1)}
-                    >
-                        <button onClick={updateLevel}> Next Level</button>
-                    </Link>
-                </div>
+                {success ? (
+                        <div className="popup-content">
```
Comment (hollan2) [inline Isle of Trust/src/components/EndOfLevel.tsx:25]: this pop up has scroll bars on each axis which is a little weird. If you can figure out how to remove that, that'd be great but if it's too hard don't worry about it

File: Isle of Trust/src/routes/TutorialGameView.tsx [hunk 3]
```diff
@@ -577,17 +577,37 @@ class TutorialView extends React.Component<StartInfo, GameViewState> {
 
     //Determines when to render the EndOfLevel component based on the level's stageCount or turnCount
     renderEndOfLevel = () => {
+
+
+        const checkResource = this.state.map.getVertices();
+
+       /* for (let i = 0; i <= checkResource.length; i++){
```
Comment (hollan2) [inline Isle of Trust/src/routes/TutorialGameView.tsx:584]: remove comment

File: Isle of Trust/src/components/EndOfLevel.tsx [hunk 4]
```diff
@@ -1,35 +1,66 @@
 import React from "react";
 import ReactDom from "react-dom";
 import { Link } from "react-router-dom";
+import "../css/EndOfLevel.css"
+import Confetti from 'react-confetti'
 
 //Renders the pop up window that appears after the level has been completed
 //TODO: This is a basic set up of the pop up using the logic from the "History" pop up, it still needs content
```
Comment (hollan2) [inline Isle of Trust/src/components/EndOfLevel.tsx:8]: should we remove these comments?

**Review by hollan2 — CHANGES_REQUESTED** [review]
Overall it looks really good! Might consider adding a delay before it pops up so we can see the result to our last round of play. Also, maybe we should list the resources of everyone too in the end card????

**Review by hollan2 — CHANGES_REQUESTED** [review]
Overall it looks really good! Might consider adding a delay before it pops up so we can see the result to our last round of play. Also, maybe we should list the resources of everyone too in the end card????

**Review by adamstag20 — COMMENTED** [review]
I have updated these changes

**Review by hollan2 — APPROVED** [review]
Looks amazing!!

**Pull request merged**